filebeat/docs/howto/migrate-to-filestream.asciidoc

```diff
@@ -0,0 +1,243 @@
+[[migrate-to-filestream]]
+== Migrate log input configurations to filestream
+
+Filestream input is generally available since 7.14. So it is high time you
+migrate your existing log input configurations.  It comes with many
+improvements over the old input like configurable order for parsers and more.
```

Comment:
```suggestion
The `filestream` input has been generally available since 7.14. So it is high time you
migrate your existing `log` input configurations.  The `filestream` input comes with many
improvements over the old input, such as configurable order for parsers and more.
```